pifx/core.py

```diff
@@ -72,6 +72,41 @@ def set_state(self, selector='all',
             method='put', endpoint='lights/{}/state',
             endpoint_args=[selector], argument_tuples=argument_tuples)
 
+    def set_states(self, states=[{'selector': 'all',
```

Comment:
This default argument seems potentially error-prone. See https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

It might be better to have `states=None` and then `if states is None: states = []` in the method itself.

Reply:
@cydrobolt I have updated the method to remove the mutable default argument. I added a usage example in the docstring so users will hopefully know how to call it. Let me know if you'd like to see any other changes!